src/python/py/models/builder.py

```diff
@@ -157,6 +157,13 @@ def __init__(self, config, io_dtype, onnx_dtype, ep, cache_dir, extra_options):
             "add_offset": 0,            # Offset value for LayerNorm weight
         }
 
+        # MatMul-specific variables
+        is_lora = hasattr(config, "peft_type") and config.peft_type == "LORA"
+        self.matmul_attrs = {
+            "use_lora": is_lora,        # Use LoRA/QLoRA format
+            "use_qdq": False,           # Use QDQ format
```

Comment:
Originally, I added `use_qdq` in `self.matmul_attrs` to prepare for the QDQ work and support both QDQ and LoRA/QLoRA. However, the QDQ work was eventually added in [this PR](https://github.com/microsoft/onnxruntime-genai/pull/900).

Given this, can we replace any references to `self.use_qdq` with `self.matmul_attrs["use_qdq"]` and change the `use_qdq` entry to the below?

https://github.com/microsoft/onnxruntime-genai/blob/b3dfed65da067f96be5d9b464a3e0b26496b8621/src/python/py/models/builder.py#L41

Then, we can remove `self.use_qdq`.

Reply:
rebase did not happen properly. Fixing it